{"name": "批量登录用户", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'url'", "trace": "name = 'yujl471', pwd = 'Jg82SqEi', assertion = 9999\n\n    @pytest.mark.parametrize('name,pwd,assertion', cases, ids=ids)\n    @allure.title(\"批量登录用户\")\n    def test_login(name, pwd, assertion):\n        url = 'http://qa.yansl.com:8084/login'\n        req = {\n            \"pwd\": pwd,\n            \"userName\": name\n        }\n>       resp = request_tool.post_json(url, json=req)\n\ntest_case\\User\\test_login.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nargs = ('http://qa.yansl.com:8084/login',)\nkwargs = {'json': {'pwd': 'Jg82SqEi', 'userName': 'yujl471'}}, r = ()\n\n    def _func(*args, **kwargs):\n        r= func(*args, **kwargs)\n        request = \"-------------------request-------------\" \\\n>                 \"\\n{0}\\n{1}\\n{2}\".format(r.url, string_tool.dic_to_str(r.request.headers), r.request.body)\nE       AttributeError: 'tuple' object has no attribute 'url'\n\ntools\\report\\decorators_tool.py:11: AttributeError"}, "attachments": [{"name": "log", "source": "b79322e0-2758-44a8-aa45-cf560bdb5d28-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "name", "value": "'yujl471'"}, {"name": "pwd", "value": "'Jg82SqEi'"}, {"name": "assertion", "value": "9999"}], "start": 1602684324955, "stop": 1602684334433, "uuid": "5649bdea-d75c-4697-bf5d-076de382b3c9", "historyId": "40f98ef4a19a9401a5adf02e55de2196", "fullName": "test_case.User.test_login#test_login[用户名错误]", "labels": [{"name": "parentSuite", "value": "test_case.User"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-679Q8H2"}, {"name": "thread", "value": "9868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.User.test_login"}]}